### PR TITLE
Update ESLint configuration and refactoring

### PR DESCRIPTION
**Pull request by mikhail-vl**

### CODE REVIEW

**Review by vitPinchuk — APPROVED** [review]
Looks good.

@asimonok ready for review

File: src/constants/toolbar.ts [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
 /**
  * Buttons Options
  */
-export const ButtonsOptions = [
-  { value: ButtonType.DOWNLOAD, label: 'Download' },
-  { value: ButtonType.NAVIGATION, label: 'Navigation' },
-  { value: ButtonType.ZOOM, label: 'Zoom' },
+export const BUTTONS_OPTIONS: Array<SelectableValue<ButtonType[]>> = [
+  { value: [ButtonType.DOWNLOAD], label: 'Download' },
```
Comment (asimonok) [inline src/constants/toolbar.ts:9]: Please remove arrays. Just `ButtonType.DOWNLOAD`

**Review by asimonok — CHANGES_REQUESTED** [review]

File: src/constants/toolbar.ts (hunk 1 above)
Comment (vitPinchuk) [reply src/constants/toolbar.ts]: updated

**Review by asimonok — APPROVED** [review]
Looks good!

**Review by asimonok — APPROVED** [review]
Looks good!

**Pull request merged**